Services/Test Markdown.R

```diff
@@ -0,0 +1,26 @@
+source("Master RMarkdown Document & Render Code/Global Script.R")
+
+return_rows <- function(x) {
+  n <- sample(x, 1)
+  return(x == n)
+}
+
+randomly_selected_localities <- read_in_localities() %>%
+  group_by(hb2019name) %>%
+  filter(return_rows(hscp_locality)) %>%
+  ungroup() %>%
+  pull(hscp_locality) %>%
+  unique()
+
+for (Loc in randomly_selected_localities) {
+  rmarkdown::render("Services/Services-Testing-Markdown.Rmd",
+    output_file = paste0("Services-Testing-Markdown-", Loc, ".docx"),
+    output_dir = "Services"
+  )
```

Comment:
This will make it knit using the project directory as root (which is what we want according to our 'convention'). This could alternatively be specified using base R with `knit_root_dir = normalizePath(".")`.

The `envir = new.env()` creates a clean environment, the same as when you click 'knit'.

```suggestion
  rmarkdown::render("Services/Services-Testing-Markdown.Rmd",
    output_file = paste0("Services-Testing-Markdown-", Loc, ".docx"),
    output_dir = "Services",
    knit_root_dir = rstudioapi::getActiveProject(),
    envir = new.env()
  )
```